### ISSUE TITLE
Inline comment combined with object annotation confuses BlockControlStructureSpacing

### ISSUE THREAD

**Issue by Dropa** [slevomat/coding-standard#1377]
With example code of

```php
    
    // Describe something.
    /** @var Object $value */
    if (!$value = $that->getSomething($from)) {
      return $somethingElse;
    }
```

`SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountBeforeControlStructure` is being triggered.

This seems like a bug, as each of these seems to be just fine;
```php
    // Describe something.
    /** @var Object $value */
    $value = $that->getSomething($from)

    if (!$value) {
      return $somethingElse;
    }
```

```php
    /** @var Object $value */

    // Describe something.
    if (!$value = $that->getSomething($from)) {
      return $somethingElse;
    }
```

```php
    // Describe something.

    /** @var Object $value */
    if (!$value = $that->getSomething($from)) {
      return $somethingElse;
    }
```

**Comment by kukulich** [comment]
Use only one comment and you should be fine.
